pkg/motor/create_account.go

```diff
@@ -86,17 +84,17 @@ func (mtr *motorNodeImpl) CreateAccount(request mt.CreateAccountRequest) (mt.Cre
 	if err != nil {
 		return mt.CreateAccountResponse{}, fmt.Errorf("setup vault: %s", err)
 	}
-	mtr.callback.OnMotorEvent(ct.MotorCallbackMessage_MTR_VAULT_CREATED, true)
 
 	// update DID Document
+	mtr.callback.OnMotorEvent("Updating DIDDocument for Account", false)
```

Comment:
References to the value represented by each string should be made through some kind of strongly typed value / constant